src/components/table.tsx

```diff
@@ -5,15 +5,34 @@ export type TableRow = (string | number | VNode)[];
 export interface TableData {
   header: TableRow;
   body: TableRow[];
+  footer?: TableRow;
 }
 
+type NumberFormatter = (n: number) => string;
+
 interface TableProps {
   data: TableData;
-  numberFormatter?: (n: number) => string;
+  numberFormatter?: NumberFormatter;
+}
+
+function renderRow(row: TableRow, numberFormatter: NumberFormatter) {
```

Comment:
Maybe it can be a proper component `TableRow` ?

Reply:
updated in  a185371dc8eb06e4b8af37f46dabb2a9f0f0de90